<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="subtractButton">
        <property name="text">
         <string>Subtract</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="multiplyButton">
        <property name="text">
         <string>Multiply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="divideButton">
        <property name="text">
         <string>Divide</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="text">
       <string>Result: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
